<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mistletoe</class>
 <widget class="EtMainWindow" name="Mistletoe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mistletoe</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>670</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="codeBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>671</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Code</string>
    </property>
    <widget class="EtListWidget" name="studentFileList" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>651</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="EtLabel" name="studentDragLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>DRAG STUDENT FILES HERE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="EtListWidget" name="baseFileList" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>651</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="EtLabel" name="baseDragLabel">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>80</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>DRAG BASE FILES HERE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="studentLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Student Files</string>
     </property>
    </widget>
    <widget class="QLabel" name="baseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Base Files</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="fileSettingsBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Find Files</string>
    </property>
    <widget class="QLineEdit" name="filterEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*.c *.cpp *.h *.cs *.java *.js *.zip</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ignoreEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>stdafx</string>
     </property>
    </widget>
    <widget class="QLabel" name="filterLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Added File Filter</string>
     </property>
    </widget>
    <widget class="QLabel" name="ignoreLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ignored Files</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dirCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Use Directories</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="runCheckBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run After Added</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addStudentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Student Files</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearStudentButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear Student Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBaseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Add Base Files</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBaseButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear Base Files</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="executionBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Commands</string>
    </property>
    <widget class="QComboBox" name="languageBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>65</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>cc</string>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>a8086</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ada</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ascii</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>c</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>csharp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fortran</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>haskell</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>java</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>javascript</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lisp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>matlab</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mips</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ml</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>modula2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pascal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>perl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>plsql</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prolog</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>python</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>scheme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>vb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>vhdl</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="ignoreCountSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="commentEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="languageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QLabel" name="ignoreCountLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ignore Count</string>
     </property>
    </widget>
    <widget class="QLabel" name="comentLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runQueryButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run Query</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveQueryButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save Query</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearQueryButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear Query</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveOutputButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save Output</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearOutputButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear Output</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>981</width>
      <height>191</height>
     </rect>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EtLabel</class>
   <extends>QLabel</extends>
   <header>etlabel.h</header>
  </customwidget>
  <customwidget>
   <class>EtMainWindow</class>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EtListWidget</class>
   <extends>QWidget</extends>
   <header>etlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
